src/goto-instrument/contracts/dynamic-frames/dfcc_instrument_loop.cpp

```diff
@@ -298,6 +298,13 @@ dfcc_instrument_loopt::add_prehead_instructions(
     code_function_callt call = library.write_set_create_call(
       addr_of_loop_write_set,
       from_integer(assigns.size(), size_type()),
+      from_integer(0, size_type()),
+      false_exprt(),
+      false_exprt(),
+      false_exprt(),
+      false_exprt(),
+      true_exprt(),
+      true_exprt(),
```

Comment:
1. Why did you have to change this one call to `write_set_create_call`?
2. It seems that, after this change, there is no further call to the 3-parameter version of `write_set_create_call`. Can you please remove it?

Reply:
This change is not necessary for this PR. removed from this PR.